BranchMetrics/ios-branch-deep-linking-attribution
# Add public API to route requests to EU endpoints

## Reference
SDK-2045 Add public API to route requests to EU endpoints

## Summary
Add public API to route requests to EU endpoints

## Motivation
Currently, clients must edit source code to use EU endpoints. This change adds a simpler public API to send requests to EU endpoints.

## Type Of Change


## Testing Instructions
Confirm setting this prior to initSession results in calls going the EU endpoint. Note apps that are not enabled for the EU endpoints, will see server errors.









cc @BranchMetrics/saas-sdk-devs for visibility.
